### DISCUSSION TITLE
Airflow migration from 1.10 to 2.7 having issue

### DISCUSSION THREAD

**Discussion by prashantkhadatkar** [apache/airflow#35528] [General]
Hi, 
We have recently migrated Airflow form version 1.10 to 2.7. After setting connections and all other things, we observe that DAG's are not showing up. 
Can someone tell me any possible reason?
Thanks

**Comment by hbc-acai** [comment]
You jumped major version. A lot of things are expected to change.

**Reply by prashantkhadatkar** [reply]
Thanks lot for reply. Could you please assist me to know what I need to change more?

**Reply by potiuk** [reply]
You are about 3 years late to the game. This is how long Airflow 1.10 is End of life. Most people who knew something about it are either gone from here or don't remember anything. Your best bet is look at this document which was written 4 years ago https://airflow.apache.org/docs/apache-airflow/stable/howto/upgrading-from-1-10/index.html - maybe it will be helpful not sure because I don't remember things from 1.10 either

Also you can go to Airflow Slack and there is an AskAstro AI chatbot (in `#airflow-astronomer` channel) that harvesta a lot of historical data and in a sense it has better memory than anone here. It might be helpful if you ask it the right questions there.

That would be my advice to you.
